PoGo.NecroBot.CLI/Config/Translations/translation.id.json

```diff
@@ -334,19 +334,19 @@
     },
     {
       "Key": "maxItemsCombinedOverMaxItemStorage",
-      "Value": "[Konfigurasi Salah] Total barang maksimum kamu (ball+potion+revive={0}) melebihi kapasitas tas kamu ({1})"
+      "Value": "[Konfigurasi Salah] Total barang maksimum Anda (ball+potion+revive={0}) melebihi kapasitas tas Anda ({1})"
     },
     {
       "Key": "recyclingQuietly",
-	  "Value": "Mendaur ulang senyap..."
+	  "Value": "Daur ulang dengan santai..."
```

Comment:
Yeah, but it makes a misleading translation though...

`Daur ulang dengan santai` -> `Relax recycling`, what does that even mean?